sdk/textanalytics/Azure.AI.TextAnalytics.Legacy.Shared/src/Custom/DocumentSentimentValue.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
```

Comment:
I was unable to get the generator transforms to prevent emitting the parameter-related enumerations as public.   @maririos had some suggestions that I was unable to get to work;  I'll be opening an issue for us to revisit this and see if we can get the generator to play nice.

For now, I believe this will suffice as we know there will not be any large changes to the REST API going forward.